app/src/main/java/org/simple/clinic/summary/PatientSummaryScreenUi.kt

```diff
@@ -5,10 +5,9 @@ interface PatientSummaryScreenUi {
   fun showEditButton()
   fun showDiabetesView()
   fun hideDiabetesView()
-  fun showContactDoctorButtonTextAndIcon()
-  fun showContactButtonProgress()
-  fun enableContactDoctorButton()
-  fun disableContactDoctorButton()
   fun showContactDoctorButton()
   fun hideContactDoctorButton()
+  fun contactDoctorButtonEnabled()
+  fun contactDoctorButtonDisabled()
+  fun contactDoctorButtonInProgress()
```

Comment:
Is this name change necessary? The older names seem more appropriate to me since it is still up to the UI to change the button state.

Reply:
Previously we had multiple methods from showing/hiding button, showing/hiding text and icon or progress bar and enabling/disabling the button itself. Since we no longer have to handle all of those cases separately I changed the method names to represent the `ProgressMaterialButton` states.

Reply:
Right, I think my thoughts were more along the lines of the phrasing of the method names.

Most methods fall into three categories:
1. Querying something for information
2. Transforming data from one form to another
3. Changing the state in some way

Which category a method falls into should be evident from reading the name. Here, the methods seem to fall into category 3 since they are changing the state of the UI, but the name does not make it clear. In fact, from reading the name, you cannot tell what category the method falls into at all.

Reply:
Ah right. I will rename the methods to better reflect that.

Reply:
Renamed, hopefully these makes sense. wdyt?
```kt
fun enableContactDoctorButton()
fun disableContactDoctorButton()
fun fetchingTeleconsultInfo()
```

Reply:
These look good, let's go ahead.